documentstore_migracao/export/sps_package.py

```diff
@@ -573,12 +573,57 @@ def _move_appendix_from_body_to_back(self):
                 for app_group_tag in app_group_tags:
                     back.append(app_group_tag)
 
+    def _move_acknowledgements_from_body_to_back(self):
+        """Move os agradecimentos do body para o respectivo back.
+        Essa identificação é feita pelo cabeçalho de agradecimentos, nos idiomas
+        oficiais do SciELO. Caso encontre o cabeçalho no body, retira o cabeçalho e move
+        o primeiro parágrafo seguinte contendo algum texto.
+        """
+
+        def _get_acknowledgements(header, body):
+            lc = header.lower()
+            ack_tags_title = body.xpath(
+                f'*[contains(translate(., "{lc}", "{header}"), "{header}")]'
+            )
+            if len(ack_tags_title) > 0:
+                text = " ".join([tag_text for tag_text in ack_tags_title[0].itertext()])
+                if len(text.strip().split()) == 1:
+                    return ack_tags_title[0]
```

Comment:
Acho que isso aqui deveria ficar dentro de um for, iterável por todos os items ack_tags_title encontrados.

Reply:
Imagina que vc tem um XML assim:

```xml
<body>
              <p><bold>Subtítulo do Texto</bold></p>
                <p></p>
                <p>O agradecimento é uma expressão de reconhecimento de uma ação.</p>
                <p>Texto de teste.</p>
                <p>AGRADECIMENTO</p>
                <p></p>
                <p>Ao Fundo de Amparo à Pesquisa</p>
</body>
```

Reply:
@robertatakenaka, feitos ajustes. Pode verificar?